nix/generate-ec2-properties.py

```diff
@@ -1,10 +1,9 @@
-#! /usr/bin/env python2
+#!/usr/bin/env python
 import json
-from pprint import pprint
 
 # FIXME: AWS support adviced against the use of this index file for anything other than pricing
-# This file also do not provide a way to confidently check if an instance use nvme or not
-# and there is currently no API that can provide such information. So manuall fixes is needed
+# This file also does not provide a way to confidently check if an instance use nvme or not
+# and there is currently no API that can provide such information. So manual fixes are needed
 # after running this script.
 # A feature request is in place to provide an endpoint that can cover our need.
 # curl -O https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/AmazonEC2/current/index.json
```

Comment:
Script still works fine on python2.  After curling this index.json file (which is 1.3 GB!!!):

```console
$ nix-shell -p python2 --run './generate-ec2-properties.py index.json'
  ...
  "x1e.xlarge" = { cores = 4; memory = 124928; allowsEbsOptimized = false; supportsNVMe = false;};
  "z1d.12xlarge" = { cores = 48; memory = 393216; allowsEbsOptimized = false; supportsNVMe = false;};
  "z1d.2xlarge" = { cores = 8; memory = 65536; allowsEbsOptimized = false; supportsNVMe = false;};
  "z1d.3xlarge" = { cores = 12; memory = 98304; allowsEbsOptimized = false; supportsNVMe = false;};
  "z1d.6xlarge" = { cores = 24; memory = 196608; allowsEbsOptimized = false; supportsNVMe = false;};
  "z1d.large" = { cores = 2; memory = 16384; allowsEbsOptimized = false; supportsNVMe = false;};
  "z1d.metal" = { cores = 48; memory = 393216; allowsEbsOptimized = false; supportsNVMe = false;};
  "z1d.xlarge" = { cores = 4; memory = 32768; allowsEbsOptimized = false; supportsNVMe = false;};
}
```